multi-node/aws/pkg/cluster/cloud_config_controller.go

```diff
@@ -14,11 +14,31 @@ coreos:
     initial-advertise-peer-urls: http://$private_ipv4:2380
     listen-client-urls: http://0.0.0.0:2379
     listen-peer-urls: http://0.0.0.0:2380
-    initial-cluster: controller=http://$private_ipv4:2380
 
   units:
+  - name: etcd-peers.service
```

Comment:
It also seems like using the ASG would be racy, when we need absolute determinism to bootstrap this cluster. How does this work that makes it safe to run in parallel on several machines?